src/tcpip_checksum/jbuild

```diff
@@ -0,0 +1,30 @@
+(library
+ ((name        tcpip)
+  (public_name tcpip)
+  (modules (tcpip_checksum))
+  (libraries   (cstruct))
+  (wrapped false)))
+
+(library
+ ((name tcpip_xen)
+  (public_name tcpip.xen)
+  (libraries (tcpip))
+  (modules (tcpip_xen))
+  (c_names (checksum_stubs_xen))
+  (c_flags (:include c_flags_xen.sexp))
```

Comment:
Copied from the quick start: http://jbuilder.readthedocs.io/en/latest/quick-start.html?highlight=c_flags.sexp#defining-a-library-with-c-stubs-using-pkg-config

FWIW I think this works with `io-page-xen` -- I put an `#error "woo"` in some code guarded by `#ifdef MINIOS` and it triggered as expected